_component_design/table-variations.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Table Variations
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Depending on your use case, you can consider adding a dual-line or dual-layout table to the page.
+preview-image: preview-images/popover.svg
+resource: true
+status: stable
+last-modified: 2018-11-19
+helix-ui-css: true
+helix-ui-javascript: true
+pagelink: https://rackerlabs.github.io/helix-ui/components/popovers/
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Table variations
+
+Note to Chris and Bart: This section will include content taken from the Table page.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the following guidelines when you consider which table variation to construct.
+
+- Use a *multi-line* table when the text in a cell wraps to two or more lines. For example, use a dual-line table when two values are meaningfully related to each other and should be displayed in the same cell.
+
+- Use a *dual-layout* table when you need to display data in two tables, one on the left and one on the right.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct table variations.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Multi-line table best practices
+
+Consider the following best practices when you construct a dual-line table:
+
+- In lines displaying two independent text rows, emphasize the primary line (on top) by increasing its font weight and adding **bold**.
```

Comment:
WTU - use dual line when you don't need to sort on data included in both lines independently. If need to sort on both, then need to have two columns.